docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+    .and(Cypher.lt(movieNode.property("released"), new Cypher.Param(2000)))
+```
+
+This takes the same parameters as `.where`, and makes sure to properly concatenate the filters with `AND`. In this case, the function `Cypher.lt` will add the operator `<` (**l**ower **t**han). Same as before, the comparison happens between a property of `movieNode` and a param. This time the param is a number. Cypher Builder will take care of correctly passing it to the `params` object.
+
+Running the script again:
+
+```cypher
+MATCH (this0:Movie)
+WHERE (this0.title = $param0 AND this0.released < $param1)
+RETURN this0
+```
+
+```javascript
+{ param0: 'The Matrix', param1: 1999 }
+```
+
+This should look closer to the target query, note that now there are two parameters, each one correctly associated to the correct filter.
+
+[NOTE]
+====
+Adding the `AND` operation automatically adds parenthesis to ensure operation precedence, this will be important when dealing with complex and nested filters 
+====
+
+== Projection
+
+Usually, returning the full node is not desirable, for these cases, you can explicitly set a projection in the `RETURN` clause.
+
+You can define multiple columns to return in the projection by passing them into the `.return` method. This method not only accepts Nodes, but any kind of variables, such as properties or params.
+
+Let's update the return subclause:
+
+```javascript
+    .return(movieNode.property("title"), movieNode.property("tagline"), movieNode.property("year"));
+```
+
+The returned Cypher now should be:
+
+```cypher
+MATCH (this0:Movie)
+WHERE (this0.title = $param0 AND this0.released < $param1)
+RETURN this0.title, this0.tagline, this0.released
+```
+
+Success! You have now a query with filters and projections.
+
+=== Reusing variables
+
+Right now, this query is filtering and returning the properties `id` and `title` of the Movie nodes. However, the query relies on the name of these properties to remain the same. This, particularly with more complex queries, can lead to flaky queries, as it is hard to keep track of strings identifying values.
+
+To fix this, you'll make a small adjustment to the script, so that both the filters and the projection use the same property variable:
+
+```javascript
+const titleProp = movieNode.property("title");
+const yearProp = movieNode.property("released");
+const taglineProp = movieNode.property("tagline");
+
+const clause = new Cypher.Match(movieNode)
+    .where(Cypher.eq(titleProp, new Cypher.Param("The Matrix")))
+    .and(Cypher.lt(yearProp, new Cypher.Param(2000)))
+    .return(titleProp, taglineProp, yearProp);
+```
+
+This change, not only ensures that different parts of the query are always in sync but also makes the clause itself shorter and easier to read.
+
+[TIP]
+====
+Params can also be assigned to a variable and reused, this can be particularly useful when having multiple filters over the same parameter.
```

Comment:
This leads me to a question whether we should use parameter or param when referring to this topic here.

```suggestion
Params can also be assigned to a variable and be reused. 
This can be particularly useful when having multiple filters over the same parameter.
```